frontend/src/shared/components/tasque-input/input.component.sass

```diff
@@ -0,0 +1,58 @@
+@import 'src/colors.sass'
+
+.input-container
+    position: absolute
```

Comment:
It is really needed to use position: absolute?

Reply:
Fixed